src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -218,26 +219,26 @@ public void deleteNetworkModifications(UUID groupUuid, List<UUID> modificationsU
 
     @Transactional
     public Optional<NetworkModificationResult> moveModifications(UUID groupUuid, UUID originGroupUuid,
-                                                                 UUID before, UUID networkUuid, String variantId,
+                                                                 UUID beforeModificationUuid, UUID networkUuid, String variantId,
                                                                  ReportInfos reportInfos, List<UUID> modificationsToMove,
                                                                  boolean canBuildNode) {
-        List<ModificationInfos> movedModifications = networkModificationRepository.moveModifications(groupUuid, originGroupUuid, modificationsToMove, before)
-            .stream()
-            .filter(m -> !m.getStashed())
-            .map(networkModificationRepository::getModificationInfos)
-            .collect(Collectors.toList());
+        List<ModificationEntity> movedMetadataEntities = networkModificationRepository.moveModifications(groupUuid, originGroupUuid, modificationsToMove, beforeModificationUuid);
```

Comment:
All the modifications corresponding to the groupUuid should be returned here to perform the application correctly. I see 2 options: 
- We return everything but then I would rater return the ModificationInfos directly
- We don't return anything and the application part of this method handles directly the application, so it retrieves ModificationInfos from groupUuid and then apply them if necessary 

What do you think ?

Reply:
as said before, this PR implements the right behaviour